code/datums/components/pricetag.dm

```diff
@@ -19,7 +20,8 @@
 
 /datum/component/pricetag/proc/Unwrapped()
 	SIGNAL_HANDLER
-
+	if(istype(parent, /obj/item/bounty_cube))
+		return
```

Comment:
Wouldn't it be easier to just not register this?

Reply:
i agree, this is a hack

Reply:
more specifically, i think components should have the ability to apply to anything aside from actual technical limitations, not design ones.

the bounty cube itself should tell the price tag component not to register it